examples/rollup/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "module": "es2015",
     "moduleResolution": "node",
     "noEmitOnError": true,
-    "suppressImplicitAnyIndexErrors": true,
+    "noUncheckedIndexedAccess": false,
```

Comment:
I don't remember why I added `suppressImplicitAnyIndexErrors` in the tsconfig.json files in the `examples` folder, but it seems ts files compile without this option. Can you just remove the option without adding `noUncheckedIndexedAccess`? It seems unnecessary.